src/input_common/udp/client.cpp

```diff
@@ -168,12 +170,22 @@ void Client::OnPadData(Response::PadData data) {
     packet_sequence = data.packet_counter;
     // TODO: Check how the Switch handles motions and how the CemuhookUDP motion
     // directions correspond to the ones of the Switch
-    Common::Vec3f accel = Common::MakeVec<float>(data.accel.x, data.accel.y, data.accel.z);
-    Common::Vec3f gyro = Common::MakeVec<float>(data.gyro.pitch, data.gyro.yaw, data.gyro.roll);
+    const auto now = std::chrono::system_clock::now();
+    u64 time_difference =
+        std::chrono::duration_cast<std::chrono::microseconds>(now - last_motion_update).count();
+    last_motion_update = now;
+    Common::Vec3f rawgyroscope = {data.gyro.pitch, data.gyro.roll, -data.gyro.yaw};
```

Comment:
```suggestion
    Common::Vec3f raw_gyroscope = {data.gyro.pitch, data.gyro.roll, -data.gyro.yaw};
```

All variables here should be const if they're able to be.